### PR TITLE
add: homepage

### PR DESCRIPTION
**Pull request by Varu98**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,14 +2,128 @@
...
+
+                <!-- nav Menu Class-->
+                <div class="nav-menu">
+                    <button class="login-btn">Login</button>
+                    <span class="icon-container">
+                        <span class="nav-icon">
+                            <ion-icon name="heart-circle-outline"></ion-icon>
+                            <span class="icon-btn__badge">8</span>
+                        </span>
+                        <a class="icon-link">Wishlist</a>
```
Comment (KaustubhM1997) [inline index.html:45]: As the icons already specify the purpose, you can skip writing Wishlist and cart. Rest looks good!

**Review by KaustubhM1997 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (Varu98) [reply index.html:45]: right will give it a try

**Review by MarkVed17 — APPROVED** [review]
Looks good, Varun!
I assume you’ll handle responsiveness later.

File: index.html [hunk 2]
```diff
@@ -2,14 +2,128 @@
...
+            <div class="trending-accessories-img-container">
+                <img class="trending-accessories-img" src="./assets/accessory/31bBMCBWCVL._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/519uMzMXJVL._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/51spbABphFL._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/61+Lfxgog+S._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/61DKcGr12OL._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/61gscZYmaoL._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/61vtLhO6fDL._AC_UL320_.jpg" alt="" />
+                <img class="trending-accessories-img" src="./assets/accessory/71lVwl3q-kL._AC_UL320_.jpg" alt="" />
+            </div>
```
Comment (MiheerTamkhane) [inline index.html:119]: If you not gonna use this commented part, clean up.

**Review by sakshamAK — APPROVED** [review]
Nice work

**Review by MiheerTamkhane — APPROVED** [review]
Look good to me

File: index.html (hunk 2 above)
Comment (Varu98) [reply index.html:119]: I need them. 👍🏻

**Pull request merged**